gramps/gui/editors/displaytabs/surnametab.py

```diff
@@ -122,10 +122,10 @@ def build_columns(self):
         renderer.connect('edited', self.on_orig_edited, self._column_combo[3])
         # add to treeview
         column = Gtk.TreeViewColumn(name, renderer, text=self._column_combo[3])
-        column.set_resizable(True)
+        column.set_resizable(False)
```

Comment:
The name origin column is no longer easily resizable.  This may be annoying for long text in translations or custom entries.